tests/includes/wait-for.sh

```diff
@@ -46,14 +46,13 @@ idle_condition() {
 }
 
 idle_subordinate_condition() {
-    local name parent unit_index parent_index
+    local name parent unit_index
 
     name=${1}
     parent=${2}
     unit_index=${3:-0}
-    parent_index=${4:-0}
 
-    path=".[\"$parent\"] | .units | .[\"$parent/$parent_index\"] | .subordinates | .[\"$name/$unit_index\"]"
+    path=".[\"$parent\"] | .units | .[] | .subordinates | .[\"$name/$unit_index\"]"
```

Comment:
So this will work for all unit subordinates, I wonder if that's what we want. Or we should have `all_idle_subordinate_condition` or some such.

Reply:
The function still expects the parent name and the subordinate name in order to work. What we are saying here is that we are finding the specific subordinate unit, not matter which of the parent _units_ it happens to be associated with.